apps/admin-portal/src/api/identity-provider.ts

```diff
@@ -18,7 +18,12 @@
 
 import { AxiosHttpClient } from "@wso2is/http";
 import { GlobalConfig, ServiceResourcesEndpoint } from "../configs";
-import { HttpMethods, IdentityProviderListResponseInterface, IdentityProviderResponseInterface } from "../models";
+import {
+    FederatedAuthenticatorListItemInterface, FederatedAuthenticatorMetaInterface,
+    HttpMethods, IdentityProviderInterface,
+    IdentityProviderListResponseInterface,
+    IdentityProviderResponseInterface
```

Comment:
```suggestion
    FederatedAuthenticatorListItemInterface,
    FederatedAuthenticatorMetaInterface,
    HttpMethods,
    IdentityProviderInterface,
    IdentityProviderListResponseInterface,
    IdentityProviderResponseInterface
```

Use new lines for all when need to go multiline